--- A/config.def.h	2015-05-23 16:37:15.777186938 -0400
+++ B/config.def.h	2015-05-23 16:50:17.823856245 -0400
@@ -1,16 +1,20 @@
 /* See LICENSE file for copyright and license details. */
 
+#define MINECRAFTC "net-minecraft-bootstrap-Bootstrap"
+
 /* appearance */
-static const char font[]            = "Sans 8";
+static const char font[]            = "Sans 8";
 static const char normbordercolor[] = "#444444";
-static const char normbgcolor[]     = "#222222";
-static const char normfgcolor[]     = "#bbbbbb";
-static const char selbordercolor[]  = "#005577";
-static const char selbgcolor[]      = "#005577";
-static const char selfgcolor[]      = "#eeeeee";
-static const unsigned int borderpx  = 1;        /* border pixel of windows */
+static const char normbgcolor[]     = "#679816";
+static const char normfgcolor[]     = "#3C3C3C";
+static const char selbordercolor[]  = "#679816";
+static const char selbgcolor[]      = "#F2F2F2";
+static const char selfgcolor[]      = "#3C3C3C";
+static const unsigned int borderpx  = 0;        /* border pixel of windows */
 static const unsigned int gappx     = 6;        /* gap pixel between windows */
 static const unsigned int snap      = 32;       /* snap pixel */
+static const unsigned int systrayspacing = 2;   /* systray spacing */
+static const Bool showsystray       = True;     /* False means no systray */
 static const Bool showbar           = True;     /* False means no bar */
 static const Bool topbar            = True;     /* False means bottom bar */
 static const Bool statusmarkup      = True;     /* True means use pango markup in status message */
@@ -19,9 +23,16 @@
 static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
 
 static const Rule rules[] = {
-	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            True,        -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       False,       -1 },
+	/* class              instance    title       tags mask     isfloating   monitor */
+	{ "Gimp",             NULL,       NULL,       NULL,              True,        -1 },
+	{ "Transmission-gtk", NULL,       NULL,       1 << 4,            False,       -1 },
+	{ "Thunderbird",      NULL,       NULL,       1 << 5,            False,       -1 },
+	{ "Hexchat",          NULL,       NULL,       1 << 6,            False,       -1 },
+	{ "Steam",            NULL,      "Steam",     1 << 7,            False,       -1 },
+	{ "Steam",            NULL,      "Friends",   1 << 7,            False,       -1 },
+	{ "Firefox",          NULL,       NULL,       1 << 8,            False,       -1 },
+	{ "qemu-system-i386", NULL,       NULL,       NULL,              True,        -1 },
+	{ MINECRAFTC,         NULL,       NULL,       NULL,              True,        -1 },
 };
 
 /* layout(s) */
@@ -37,7 +48,7 @@
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
@@ -48,13 +59,29 @@
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
-static const char *dmenucmd[] = { "dmenu_run", "-fn", font, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
-static const char *termcmd[]  = { "uxterm", NULL };
+static const char *dmenucmd[]   = {"dmenu_run", "-fn", font, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
+static const char *termcmd[]    = {"lxterminal"};
+static const char *vtogglecmd[] = {"amixer", "-q", "sset", "Master", "toggle"};
+static const char *vdowncmd[]   = {"amixer", "-q", "sset", "Master", "3%-"};
+static const char *vupcmd[]     = {"amixer", "-q", "sset", "Master", "3%+"};
+static const char *mtoggle[]    = {"bmpc", "toggle"};
+static const char *mstop[]      = {"bmpc", "stop"};
+static const char *mprev[]      = {"bmpc", "prev"};
+static const char *mnext[]      = {"bmpc", "next"};
+static const char *screenshot[] = {"screenshot"};
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
+	{ MODKEY,                       XK_F1,     spawn,          {.v = vtogglecmd } },
+	{ MODKEY,                       XK_F2,     spawn,          {.v = vdowncmd } },
+	{ MODKEY,                       XK_F3,     spawn,          {.v = vupcmd } },
+	{ MODKEY,                       XK_F5,     spawn,          {.v = mtoggle } },
+	{ MODKEY,                       XK_F6,     spawn,          {.v = mstop } },
+	{ MODKEY,                       XK_F7,     spawn,          {.v = mprev } },
+	{ MODKEY,                       XK_F8,     spawn,          {.v = mnext } },
+	{ MODKEY,                       XK_s,      spawn,          {.v = screenshot } },
 	{ MODKEY,                       XK_b,      togglebar,      {0} },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
@@ -104,4 +131,3 @@
 	{ ClkTagBar,            MODKEY,         Button1,        tag,            {0} },
 	{ ClkTagBar,            MODKEY,         Button3,        toggletag,      {0} },
 };
-
